sda/cmd/ingest/ingest.go

```diff
@@ -253,11 +253,11 @@ func (app *Ingest) ingestFile(correlationID string, message schema.IngestionTrig
 		if err != nil {
 			switch {
 			case strings.Contains(err.Error(), "no such file or directory") || strings.Contains(err.Error(), "NoSuchKey:"):
-				log.Errorf("Failed to open file to ingest reason: (%s)", err.Error())
+				log.Errorf("Failed to open file to ingest, file-id: %s, reason: (%s)", fileID, err.Error())
```

Comment:
```suggestion
				log.Errorf("Failed to open file to ingest, file-id: %s, inbox path: %s, reason: (%s)", fileID, message.FilePath, err.Error())
```
Here we want the file apth as well so we don't have to go into the DB when we start debugging.